js/renderingPicture.js

```diff
@@ -9,6 +9,7 @@ const createPicture = (data) =>{
   randomPhoto.alt = data.description;
   sampleClone.querySelector('.picture__likes').textContent = data.likes;
   sampleClone.querySelector('.picture__comments').textContent = data.comments.length;
+  randomPhoto.dataset.indexId = data.id;
```

Comment:
я убрал index из index-id но если хочешь оставь, ну и в методе верни тоже
```suggestion
  randomPhoto.dataset.id= data.id;
```